pkgs/development/libraries/science/math/sympow/default.nix

```diff
@@ -0,0 +1,82 @@
+{ stdenv
+, fetchurl
+, fetchpatch
+, makeWrapper
+}:
+
+stdenv.mkDerivation rec {
+  pname = "sympow";
```

Comment:
You don't need pname if you're only using it to set the ```name```.

Reply:
Its more of a style choice, I prefer to have separate attributes for name and version. But I can understand that inconsistency might be a negative here. Do you want me to remove `pname`?

Reply:
The standard way of doing this is not to define pname unless it's actively used for something.